### PR TITLE
Lower test logger max path limit to allow wiggle room for build syste…

### PR DESCRIPTION
**Pull request by BrennanConroy**
…ms copying artifacts

@muratg We want this for 2.1+

### CODE REVIEW

**Comment by pakrym** [comment]
Let's try it.

File: src/Microsoft.Extensions.Logging.Testing/AssemblyTestLog.cs [hunk 1]
```diff
@@ -21,7 +21,7 @@ public class AssemblyTestLog : IDisposable
     {
         public static readonly string OutputDirectoryEnvironmentVariableName = "ASPNETCORE_TEST_LOG_DIR";
         private static readonly string LogFileExtension = ".log";
-        private static readonly int MaxPathLength = 245;
```
Comment (JunTaoLuo) [inline src/Microsoft.Extensions.Logging.Testing/AssemblyTestLog.cs]: Hmmm maybe make this configurable via an env var.

Comment (BrennanConroy) [reply src/Microsoft.Extensions.Logging.Testing/AssemblyTestLog.cs]: That could be good

**Comment by BrennanConroy** [comment]
Added an environment variable

**Pull request merged**